scanner/cmd/updater/main.go

```diff
@@ -4,23 +4,49 @@ import (
 	"context"
 	"flag"
 	"log"
+	"time"
 
+	"github.com/pkg/errors"
 	"github.com/quay/zlog"
 	"github.com/stackrox/rox/scanner/updater"
 )
 
+func tryExport(outputDir string) error {
+	const timeout = 35 * time.Minute
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+
+	err := updater.Export(ctx, outputDir)
+	if errors.Is(ctx.Err(), context.DeadlineExceeded) {
+		zlog.Error(ctx).Err(ctx.Err()).Msg("Data export attempt failed; will attempt retry if within retry limits")
```

Comment:
`err` instead of `ctx.Err()`. Also you can just say "Export timed out". In theory, `tryExport` doesn't know anything about the potential to retry